README.md

```diff
@@ -67,6 +67,7 @@ Example:
 Refer to [Create a simple API](http://loopback.io/doc/en/lb3/Create-a-simple-API.html) 
 for more information.
 
+Refer to [README](./soap/README_Soap.md) for more information on 'lb soap' command.
```

Comment:
If find it confusing when a README files contains a reference to README again. My proposal:

```diff
- Refer to [README](./soap/README_Soap.md)
+ Refer to [README_Soap](./soap/README_Soap.md)
```